frontend/package.json

```diff
@@ -10,14 +10,19 @@
     "format.check": "prettier --check src"
   },
   "dependencies": {
+    "@emotion/react": "^11.4.1",
+    "@emotion/styled": "^11.3.0",
     "@material-ui/core": "^4.12.3",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.60",
+    "@mui/material": "^5.0.2",
+    "@types/react": "^16.8.6 || ^17.0.0",
     "dotenv": "^10.0.0",
     "parcel-bundler": "^1.12.5",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-router-dom": "^5.3.0"
+    "react-router-dom": "^5.3.0",
+    "styled-components": "^5.3.1"
```

Comment:
Can you remove the styled-components dependencies if they aren't used?

Reply:
Yes!